templates/terraform/modules/environment/main.tf

```diff
@@ -118,3 +118,12 @@ module "logging" {
   ebs_volume_size_in_gb = var.logging_volume_size_in_gb
   create_service_role   = var.logging_create_service_role
 }
+
+module "sendgrid" {
+  source = "commitdev/zero/aws//modules/sendgrid"
+  version = "0.0.2"
+  count  = var.sendgrid_enabled ? 1 : 0
+
+  zone_name = var.domain_name
+  sendgrid_api_key_secret_name = var.sendgrid_api_key_secret_name
```

Comment:
almost all the logic has been moved to https://github.com/commitdev/terraform-aws-zero/pull/1/commits/bd194a0e680a10b59c15aa6e1236d050e2d2b82d with the external-datasource 🎉